src/pages/candidate-details/candidate-details.ts

```diff
@@ -29,6 +29,11 @@ interface CandidateDetailsPageState {
   business: Business;
 }
 
+enum driverType {
+  R = 'Rider',
```

Comment:
can you use 'RIDER' and 'DRIVER' rather than 'R' and'D'

Reply:
Updated enum in line with your comments